zebra-node-services/src/scan_service/response.rs

```diff
@@ -33,5 +30,5 @@ pub enum Response {
     ClearedResults,
 
     /// Response to `SubscribeResults` request
-    SubscribeResults(mpsc::Receiver<Arc<Hash>>),
+    SubscribeResults(mpsc::Receiver<Hash>),
```

Comment:
> It might be useful to use `transaction::Hash` to distinguish the type from `block::Hash`.

Agreed, updated in https://github.com/ZcashFoundation/zebra/pull/8253/commits/b6f5461389d5af6a4c7e427f6a866b75a177b5c2.

> Do we need to return Height as well ?

Yeah, I think adding height and key is a good idea. I'll do that here.